### DISCUSSION TITLE
Has anyone run gpt-researcher with fine-tuned models?

### DISCUSSION THREAD

**Discussion by adrianhensler** [assafelovic/gpt-researcher#921] [Ideas]
I originally started thinking about fine-tuning a local model to assist directly (somehow?) with research or report creation - but fine-tuning an OpenAI GPT model seems like a better idea.

Curious if anyone has tried this (running with a fine-tuned model), and what the use case was? I'd like to think that this would provide some markedly better results, depending on the domain and training.

**Comment by ElishaKay** [comment]
Sup @adrianhensler 

Agreed. Happy to hear how it plays out.
Would be great to compile a list of sources / best practices around this topic.

**Comment by adrianhensler** [comment]
Yes, I've certainly not pre-trained any models let alone tested this tool with one. I've done some initial review. I was sort of hoping someone with more insight might be able to jump start it in at least a bit of direction that makes sense. I'll post what I've got hopefully later today or tomorrow when I get it into something a bit more sensible.

**Comment by coreylansdell** [comment]
I really want to try it. I have been trying to install it for 2 days. Just keep getting errors. Nothing has worked for me. 
I've followed several tutorials. I just keep running into errors.
